pkg/network/event_common.go

```diff
@@ -194,6 +194,8 @@ type ConnectionStats struct {
 	DNSStatsByDomainByQueryType map[*intern.Value]map[dns.QueryType]dns.Stats
 
 	Via *Via
+
+	IsAssured bool
```

Comment:
Why was this added?

Reply:
This flag comes from the tcp stats map, but is used in determining the timeout for a flow. In order to maintain an abstraction, it needed to be added to `ConnectionStats`.